src/foam/nanos/controller/ApplicationController.js

```diff
@@ -364,7 +364,7 @@ foam.CLASS({
 
       var userNotificationQueryId = this.subject && this.subject.realUser ?
       this.subject.realUser.id : this.user.id;
-
+      //is it called?
```

Comment:
is src/foam/nanos/controller/ApplicationController.js initE method or nanopay/src/net/nanopay/ui/Controller.js onUserAgentAndGroupLoaded method is called for not authorized user?